bedrock/firefox/templates/firefox/campaign/unfck/index.html

```diff
@@ -166,6 +144,28 @@ <h2 class="c-section-title">The Checklist</h2>
       src_name='un',
     ) %}{% endcall %} -->
 
+    {% set share_text='2020 might be fcked, but the internet doesn’t have to be. Here are 5 easy ways we can all help turn things around.' %}
+    {% set share_url = 'http://mzl.la/unfck' %}
+
+    {% set tw = 'https://www.twitter.com/intent/tweet?url=' + share_url|urlencode + '&text=' + share_text|urlencode + '&hashtags=unfckTheInternet' %}
+    {% set fb = 'http://facebook.com/sharer.php?u=' + share_url|urlencode + '&quote=' + share_text|urlencode %}
+
+    {% call item(
+      title='Share the internet love',
+      info='There’s a lot to unfck, but we’re powerful together. Tell your friends you browse with your values, and get them in on this conversation.',
+      alt='Click the unfck button.',
+      include_cta=True,
+      id='share',
+      src_name='button'
+    ) %}
+      <p class="c-item-cta">
+        <a id="js-tw" class="mzp-c-cta-link" href="{{ tw }}" target="_blank" rel="external noopener" data-cta-type="link" data-cta-text="Checklist: share">Tweet this</a></p>
+      </p>
+      <p class="c-item-cta">
+        <a id="js-fb" class="mzp-c-cta-link" href="{{ fb }}" target="_blank" rel="external noopener" data-cta-type="link" data-cta-text="Checklist: share">Facebook this</a></p>
```

Comment:
Not a blocker but just calling out. FaceBook Container (which the landing page promotes) gives a warning on this link: 
<img width="606" alt="image" src="https://user-images.githubusercontent.com/400117/94542617-ea7aa200-0240-11eb-91b7-0097452cdbb4.png">

Reply:
I don't think there's any way to link to Facebook without this happening.